DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CentreConfiguration/EditCentreDetailsViewModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+{
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Web.Attributes;
+    using Microsoft.AspNetCore.Http;
+
+    public class EditCentreDetailsViewModel
+    {
+        public EditCentreDetailsViewModel() { }
+
+        public EditCentreDetailsViewModel(Centre centre)
+        {
+            NotifyEmail = centre.NotifyEmail;
+            BannerText = centre.BannerText;
+            CentreSignature = centre.SignatureImage;
+            CentreLogo = centre.CentreLogo;
+        }
+
+        [Required(ErrorMessage = "Enter an email address")]
+        [MaxLength(250, ErrorMessage = "Email address must be 250 characters or fewer")]
+        [EmailAddress(ErrorMessage = "Enter an email address in the correct format, like name@example.com")]
+        [NoWhitespace("Email address must not contain any whitespace characters")]
+        public string? NotifyEmail { get; set; }
+
+        [Required(ErrorMessage = "Enter the centre support details")]
+        [MaxLength(250, ErrorMessage = "Centre support details must be 250 characters or fewer")]
+        public string? BannerText { get; set; }

```

Comment:
In my head it made more sense to keep this as BannerText and have Centre support details be a user-facing thing only but I'm open to hearing otherwise

Reply:
That sounds good to me.